public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
```

Comment:
files structure :
1- in the Prefences bar  folder.. 
u dont have component called Preferences bar so u dont need this folder ,extract the components inside and delete folder 

2- Days and Regions  folders should be named as component name